### DISCUSSION TITLE
command not found

### DISCUSSION THREAD

**Discussion by kevinsrm** [projectdiscovery/nuclei#3827] [Q&A]
I need to know which folder it was installed in

**Comment by MetzinAround** [comment]
This video might help with some common issues with new installs. 

https://www.youtube.com/watch?v=wtHcXA2l9c4

Let me know if the steps in this don't help!

**Comment by geonnave** [comment]
After following the [official instructions](https://docs.nuclei.sh/getting-started/install), I had the same problem in MacOS: `zsh: command not found: nuclei`.

The go binaries are installed in `~/go/bin`, thus one can run nuclei with `~/go/bin/nuclei`.

On the other hand, it works out of the box when installing via `brew`.

**Reply by MetzinAround** [reply]
It might be a $PATH issue. If `~/go/bin` isn't in your $PATH it won't run. 


Interesting that it works out of the box with Brew...

**Reply by MetzinAround** [reply]
![Screenshot -0918 at 08 48](https://github.com/projectdiscovery/nuclei/assets/65838556/78223cb0-de2e-4291-abfb-515df05b4abe)
This is what I had to add to my PATH to get it to work on my mac

**Reply by MetzinAround** [reply]
This is installing everything through PDTM

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
